lib/view/ui/userdatawizard/personalinfo_page.dart

```diff
@@ -65,10 +81,65 @@ class PersonalInfoPage extends StatelessWidget {
               CustomSizedBox(),
               ElevatedButton(
                 key: Key('nextButton'),
-                onPressed: () {
+                style: ElevatedButton.styleFrom(
+                  backgroundColor: Theme.of(context).colorScheme.secondary,
+                ),
+                onPressed: () async {
                   if (controller.formKey.currentState!.validate()) {
```

Comment:
break this on Press code in  funtions.

Alway break code to multple funtion, if multple lines are in same funtions